core/src/main/scala/org/apache/spark/deploy/master/Master.scala

```diff
@@ -639,10 +640,11 @@ private[deploy] class Master(
     // in the queue, then the second app, etc.
     for (app <- waitingApps if app.coresLeft > 0) {
       val coresPerExecutor: Option[Int] = app.desc.coresPerExecutor
+      val coreNumPerTask = app.desc.coresPerTask
```

Comment:
nitpick: `coreNumPerTask` sounds weird.  `coresPerTaks` is what you've used everywhere else, and the name above is `coresPerExecutor`.